api/src/utils/mailer.ts

```diff
@@ -0,0 +1,65 @@
+import nodemailer from 'nodemailer';
+import { IUser, IPitch } from 'ssw-common';
+import { pitchStatusEnum } from './enums';
+
+type message = { to: string; from: string; subject: string; html: string };
+
+export const approvedMessage = (
```

Comment:
How do you feel about abstracting this to a file called `messages` or `emailTemplates` (or something appropriate) so as we add more messages as necessary it won't clutter this util file